### ISSUE TITLE
steam prices are not listed for TR

### ISSUE THREAD

**Issue by kcyakici** [IsThereAnyDeal/IsThereAnyDeal-Issues#3388]
I don't know when this has started, but it has been a problem for some time now. I cannot see the Steam prices for games in my waitlist. I have observed that this problem may only occur in TR, I have checked EU1 and Steam prices were visible there.

**Comment by sydfouzan** [comment]
Yes. I also seem to facing the same problem. Even the waitlist shows a dollar sign next to it which makes it confusing whether it will wait for it to be lower than the dollar amount or turkish lira amount
